### ISSUE TITLE
Comparing Columns

### ISSUE THREAD

**Issue by ZIRANA11** [africanmathsinitiative/R-Instat#4374]
On the filter dialog there is only one receiver box making it impossible to compare two columns in a case where one needs to filter out some values from two different columns. In my case I wanted to find values from the column **start2** that are greater than those in **start1**, but with only one receiver box it couldn't be done.
![filter](https://user-images.githubusercontent.com/28622260/35321056-028a8278-00f7-11e8-8a56-5701f9e3b97c.PNG)

**Comment by Patowhiz** [comment]
A good feature suggested here. This reminds to also ask about editing a filter.

**Comment by rdstern** [comment]
You are right, there is a lot that still needs to be added on the system for filters.  Not easy stuff but worth considering alongside the other tasks for discussion with David/Danny.  For example there is no OR (there is AND).
Now that said, it would be good for users to also know about "work-arounds" when needed.  So how would you do this now if needed (say for E-SIAC)?  Hint - think of our powerful calculator!

**Comment by ZIRANA11** [comment]
Yes the calculator can be useful for now,
